CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+# How To Contribute
+There are several ways to contribute to the development of ReactOS. The most often encountered problem is not knowing where to begin or what to do. If you are able to program or understand the technical information that is pertinent to this project, helping the development can be easy.
+
+## Documentation
+
+There are some important points if you'd like to help document ReactOS:
+
+1.  Make sure the documentation doesn't exist yet (if it does, help improve it).
+2.  Respect [clean room reverse engineering] practices.
+3.  Add your knowledge to a place where the other developers can find it.
+
+## Test ReactOS
+
+### Localize bugs
+
+By localizing bugs, developers can identify what causes the bug and what it affects. There are a variety of methods to [debug] ReactOS while testing it. After identifying a bug, check if it is already known about by searching [JIRA] and adding any additional information to the report. If you think that it is an unidentified bug, consider [filing a bug report].
+
+### Fix bugs
+
+Instead of looking for bugs, you can also try to fix a few that are already listed in JIRA. Fixing bugs requires a lot more skill than simply searching for them, and can be time consuming.
```

Comment:
"a lot more skill" --> "a lot more skills" (terminating 's') : @gedmurphy or @jimtabor, could you confirm?

Reply:
You can either have "a lot more skill" or "many more skills" if you want plurality.
"a lot more skill" flows better in this context.